ci/build.yml

```diff
@@ -503,7 +506,8 @@ jobs:
          "is_fork": "$(System.PullRequest.IsFork)",
          "pr": "$PR_NUM",
          "pr_url": "https://github.com/digital-asset/daml/pull/$PR_NUM",
-         "pr_source_branch": "$PR_BRANCH"}
+         "pr_source_branch": "$PR_BRANCH",
+         "is_release": $(is_release)}
```

Comment:
No need for quotes here because `is_release` is either `true` or `false` and that's valid JSON.